eventuate-common-jdbc/build.gradle

```diff
@@ -1,10 +1,10 @@
 dependencies {
     compile project (":eventuate-common-json-mapper")
 
-    compile "com.fasterxml.jackson.core:jackson-databind:2.8.8"
-    compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.8.8"
-    runtime 'mysql:mysql-connector-java:5.1.39'
-    runtime ('org.postgresql:postgresql:42.2.4') {
+    compile "com.fasterxml.jackson.core:jackson-databind:2.10.2"
```

Comment:
It makes no sense to have these Jackson dependencies here. They are duplicated from json-mapper

Reply:
Oh, It seems I missed it during merge. Thank you.

Reply:
Fixed.